lib/github.js

```diff
@@ -39,6 +39,10 @@ export default {
   createStatus: async (owner, repo, head_sha, entries) => {
     const github = await getRestClient();
 
+    const rows = entries.map(entry => `|[${entry.artifactId}](${entry.url})|<code>${entry.artifactId}:incrementals;${entry.groupId};${entry.version}</code>|`).join("\n")
+
+    const table = "|Download link(s)|[Plugin Installation Manager input format](https://github.com/jenkinsci/plugin-installation-manager-tool/#plugin-input-format)|\n|--|--|\n" + rows
```

Comment:
> this seems to do for all artifacts when it will only work for plugins, is that an issue?

I could add a mention about it in the header:

```suggestion
    const table = "|Download link(s)|[Plugin Installation Manager input format](https://github.com/jenkinsci/plugin-installation-manager-tool/#plugin-input-format) (may not apply)|\n|--|--|\n" + rows
```
or
```suggestion
    const table = "|Download link(s)|[Plugin Installation Manager input format](https://github.com/jenkinsci/plugin-installation-manager-tool/#plugin-input-format) (works only for plugin)|\n|--|--|\n" + rows
```

WDYT?

Reply:
do you not have access to the maven packaging?

https://github.com/jenkinsci/pipeline-model-definition-plugin/blob/76afff0a26188ad6ced215e4663cd2115c003a74/pipeline-model-definition/pom.xml#L36

Reply:
Only the zip archive is retrieved to be uploaded to the maven repository.

I could retrieve the pom with something similar to https://github.com/jenkins-infra/incrementals-publisher/blob/f16ecec382ac18e4ebd9020ac8c99ef40a8cb5bd/lib/pipeline.js#L65-L74

then parse it to find the packaging.

Do you think it worth it?

Reply:
probably not

Reply:
I think I found a lighter solution to detect it: I just have to check if there is a .hpi file in the build metadata artifact.

Reply:
Nevermind, I was wrong: the pom is actually parsed at https://github.com/jenkins-infra/incrementals-publisher/blob/f16ecec382ac18e4ebd9020ac8c99ef40a8cb5bd/lib/permissions.js#L40-L59

Reply:
Packaging retrieved in https://github.com/jenkins-infra/incrementals-publisher/pull/88/commits/ca611e153df9f61a91331cb3c5d75f9f3160d36d, alongside the groupId and the version.

Used to display the plugin manager input format only if the entry packaging is "hpi" in https://github.com/jenkins-infra/incrementals-publisher/pull/88/commits/af2ca576a6139f88b0795755ab7d537924e938d2

Reply:
I don’t think a table is good for this as you will only have 1 plugin in almost all cases.

maybe a separate section after the links?

Reply:
@timja I've updated the output, see the "after" in the PR body.